### DISCUSSION TITLE
the vast amounts of over-allocated hard drive space link has expired

### DISCUSSION THREAD

**Discussion by bartoszgolebiowski** [Chia-Network/chia-blockchain#12082] [Support]
![image](https://user-images.githubusercontent.com/39523638/175788330-479875f9-8959-47b3-ab92-7ba25b16797e.png)
This  [link](https://idcdocserv.com/US47836321) has expired.

**Comment by loppefaaret** [comment]
thank you for the report, i will send it up the chain and get it fixed
